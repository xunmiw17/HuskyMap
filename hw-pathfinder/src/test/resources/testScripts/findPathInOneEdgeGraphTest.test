CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 3.0

FindPath simpleGraph A B