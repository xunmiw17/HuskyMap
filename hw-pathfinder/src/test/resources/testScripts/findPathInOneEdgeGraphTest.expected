created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 3.000 from A to B in simpleGraph

path from A to B:
A to B with weight 3.000
total cost: 3.000

